cli.py

```diff
@@ -161,9 +161,9 @@ def main(
         ),
         ProcessingStep(
             "download",
-            "download_all_pdfs",
+            "download_artifacts",
```

Comment:
This should be `download_all_artifacts` if it is intended to match the method name on the dataset.